assets/css/about.css

```diff
@@ -0,0 +1,355 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap');
+
+:root {
+  --primaryColor: #fca400;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+}
+
+/* Desktop version */
+nav {
+  display: flex;
```

Comment:
- [ ] Please write DRY CSS code. The line  ```display: flex;``` is repeated 13 times in your code. I suggest you have one class called ```flex```  and assign it to all elements on your page that should have the ```display: flex;``` property 👍🏾 

```flex-direction: column;``` is repeated 7 times 
``` justify-content: center;``` 8 times
``` flex-wrap: wrap;``` 3 times. There are many more repetitions, am counting on you to find them and fix them too since I cant list all of them here.
Please go through your CSS files and re-write DRY code

Reply:
ok